packages/pypy/build.sh

```diff
@@ -0,0 +1,278 @@
+TERMUX_PKG_HOMEPAGE=https://pypy.org
+TERMUX_PKG_DESCRIPTION="A fast, compliant alternative implementation of Python"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+_MAJOR_VERSION=2.7
+TERMUX_PKG_VERSION=7.3.6
+TERMUX_PKG_SRCURL=https://downloads.python.org/pypy/pypy$_MAJOR_VERSION-v$TERMUX_PKG_VERSION-src.tar.bz2
+TERMUX_PKG_SHA256=0114473c8c57169cdcab1a69c60ad7fef7089731fdbe6f46af55060b29be41e4
+TERMUX_PKG_DEPENDS="gdbm, libandroid-posix-semaphore, libandroid-support, libbz2, libcrypt, libexpat, libffi, liblzma, libsqlite, libxml2, ncurses, ncurses-ui-libs, openssl-1.1, readline, zlib"
+TERMUX_PKG_BUILD_DEPENDS="binutils, clang, dash, make, ndk-multilib, pkg-config, python2, tk, xorgproto"
+TERMUX_PKG_RECOMMENDS="clang, make, pkg-config"
+TERMUX_PKG_SUGGESTS="pypy-tkinter"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_BLACKLISTED_ARCHES="arm"
```

Comment:
What type of issue does arm have? (Please add a comment about it for the future)

Reply:
`termux-docker` doesn't provide the docker image of arm architecture. See [termux-docker: Known issues](https://github.com/termux/termux-docker#known-issues).